process/block/preprocess/validatorInfoPreProcessor.go

```diff
@@ -145,6 +151,13 @@ func (vip *validatorInfoPreprocessor) ProcessMiniBlock(miniBlock *block.MiniBloc
 		return nil, process.ErrValidatorInfoMiniBlockNotFromMeta
 	}
 
+	if vip.blockSizeComputation.IsMaxBlockSizeWithoutThrottleReached(1, len(miniBlock.TxHashes)) {
```

Comment:
We need another function in the blocksize estimator that will better handle the peer changes as those are not hashes. cc @sasurobert

Reply:
Those are taken as it is. It is hard to estimate what is the actual value

Reply:
Please add a TODO to fix the estimator. It is not complicated to do so in a separate PR.

Reply:
The good part is that until then we can reduce if needed the MaxSizeInBytes from [BlockSizeThrottleConfig] in config.toml to 85% from 90% and in this way we should avoid any kind of "message too large" from p2p